tests/security/swtpm/swtpm_uefi_verify.pm

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Ship the "swtpm" software TPM emulator for QEMU,
+#          test UEFI guest OS under libvirt, cover both
+#          TPM 1.2 and TPM 2.0
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#81256, tc#1768671
+
+use base 'opensusebasetest';
+use swtpmtest;
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+    foreach my $i ("swtpm_1", "swtpm_2") {
```

Comment:
This file is pretty much the same as `swtpm_legacy_verify.pm`. Can you merge them?

It could either do both tests or do either of them based on a `SWTPM_BOOT` (or so) variable set by either the schedule or `main_common.pm`.

Reply:
Will do